docs/src/Button.doc.js

```diff
@@ -428,42 +449,58 @@ function A11yExDisclosure() {
 
 card(
   <Example
-    id="refExample"
-    name="Example: ref"
-    description={`
-    A \`Button\` with an anchor ref to a Flyout component
-  `}
+    name="Accessibility I"
+    id="accessibilityI"
     defaultCode={`
-function ButtonFlyoutExample() {
-  const [open, setOpen] = React.useState(false);
-  const [checked, setChecked] = React.useState(false);
+function AccessibilityExample() {
+  const [isFollowing, setIsFollowing] = React.useState(false);
 
-  const anchorRef = React.useRef(null);
+  return (
+    <Button
+      accessibilityLabel={isFollowing ? "Following Alberto on Pinterest" : "Follow Alberto on Pinterestlow"}
+      onClick={() => setIsFollowing(!isFollowing)}
+      inline
+      text={isFollowing ? "Following" : "Follow"}
+    />
+  );
+}
+`}
+  />
+);
+
+card(
+  <Example
+    name="Accessibility II"
+    id="accessibilityII"
+    defaultCode={`
+function AccessibilityExample() {
+  const [isOpen, setOpen] = React.useState(false);
+  const COLLAPSIBLE_MENU = "collapsible-menu"
 
   return (
-    <>
+    <Box column={5}>
       <Button
+        accessibilityControls={COLLAPSIBLE_MENU}
+        accessibilityExpanded={isOpen}
+        accessibilityHasPopup="true"
         inline
-        color="red"
-        onClick={() => setOpen(true)}
-        ref={anchorRef}
-        text="Submit"
+        iconEnd={isOpen ? "arrow-down" : "arrow-forward"}
```

Comment:
@AlbertCarreras same here - let's check with design whether we actually want to toggle the icon

Reply:
DONE